char_counter.cpp

```diff
@@ -0,0 +1,44 @@
+#include<conio.h>
```

Comment:
Please dont use conio.h. conio.h is a C header file used in old MS-DOS compilers. It's platform specific. It is not supported on linux & lacks compatibility with compilers like Gcc.